src/auth/downscopedclient.ts

```diff
@@ -75,6 +73,13 @@ export interface CredentialAccessBoundary {
   accessBoundary: {
     accessBoundaryRules: AccessBoundaryRule[];
   };
+  /**
+   * An optional STS access token exchange endpoint.
+   *
+   * @example
+   * 'https://sts.googleapis.com/v1/token'
+   */
+  tokenURL?: string | URL;
```

Comment:
Any reason to expose this? We don't in other languages.

Reply:
Opened a PR to revert for consistency: https://github.com/googleapis/google-auth-library-nodejs/pull/1744